crates/mast-package/src/package/mod.rs

```diff
@@ -16,6 +17,60 @@ pub use self::{
 };
 use crate::MastArtifact;
 
+// PACKAGE KIND
+// ================================================================================================
+
+/// The kind of project that produced this package.
+///
+/// This helps consumers of the package understand how to use it (e.g., as an account component,
+/// a note script, a transaction script, etc.).
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
```

Comment:
For this, I think it would make sense to _not_ derive `Serialize` and `Deserialize`, and instead provide hand-written impls that expect a string representation (corresponding to the `Display` output) when the serializer/deserializer is human-readable, and expect the `u8` representation when the format is _not_ human-readable.

The default `derive` implementation of `Serialize`/`Deserialize` for enums is never ideal, so this is one place where avoiding `derive` of these traits makes sense.